text/0000-tiered-browser-support.md

```diff
@@ -0,0 +1,289 @@
+projects, but it's unclear how many parallel tests the open source plan would
+give us, and we'd at least be spending time waiting for test results on the
+various stacks. [BrowserStack also doesn't support every platform][bs-support],
+Linux on the desktop being a notable section of our userbase missing from their
+platforms.
+
+[BrowserStack]: https://www.browserstack.com/pricing
+[bs-support]: https://www.browserstack.com/support
+
+# Detailed design
+[design]: #detailed-design
+
+## Rust web content
+
+Officially produced web content includes:
+
+- rust-lang.org
+- blog.rust-lang.org
+- play.rust-lang.org
+- crates.io
+- Rustdoc output
+- thanks.rust-lang.org
+
+Explicitly not included:
+
+- Content for people working on Rust itself, such as:
+  - [The Rust Forge][]
+  - [rusty-dash][]
+
+[The Rust Forge]: https://forge.rust-lang.org/
+[rusty-dash]: https://rusty-dash.com/
+
+Things that are not really under our control but are used for official or
+almost-official Rust web content:
+
+- GitHub
+- docs.rs
+- Discourse (used for [urlo][] and [irlo][])
+- [mdBook][] output (used for the books and other documentation)
+
+[urlo]: https://users.rust-lang.org/
+[irlo]: https://internals.rust-lang.org/
+[mdBook]: https://github.com/azerupi/mdBook/
+
+## Proposed browser support tiers
+
+Based on [actual usage metrics][] and with a goal of spending our time in an
+effective way, the browser support tiers would be defined as:
+
+[actual usage metrics]: #google-analytics-browser-usage-stats
+
+### Tier 1
+
+Goal: Ensure functionality of our web content for 80% of users.
+
+Browsers:
+
+- Chrome current stable and one previous release
+- Firefox current stable and the latest ESR
+- Safari current stable, one previous minor release, and the current version on
+  iOS
+- UC Browser current stable
+
+Support:
+
+- We add automated testing for each of these as much as possible (and work on
+  adding automated testing to those web contents that aren't currently tested,
+  such as rustdoc output).
+- Bugs affecting the functionality of the sites in these browsers are
+  prioritized highly.
+
+### Tier 2
+
+Goal: Have some support of our web content, without additional time, effort, or
+monetary cost, for about 90% of users.
+
+Browsers:
+
+- Chrome up to 3 releases back and 1 release ahead of the current stable
+- Firefox up to 2 releases back and 2 releases ahead of the current stable, one
+  previous ESR
+- Safari up to 2 releases back from the current stable
+- UC Browser 1 previous stable release
+- Edge current stable and 2 previous releases
+- IE 11 until its EOL
+- Android Webview current stable and 1 previous release
+- Opera latest stable
+
+Support:
+
+- No automated testing for these.
+- Bug reports for these browsers are left open but given a low priority, and
+  closed if they are still open when a browser is old enough to drop out of
+  this tier.
+- Pull requests to fix functionality for these browsers are accepted.
```

Comment:
IMO we should not have this tier - I don't see a meaningful difference between this and tier 9999